.ci/docker/common/install_rocm_magma_docker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# TODO upstream differences from this file is into the (eventual) one in pytorch
+# - (1) check for static lib mkl
```

Comment:
cc @jithunnair-amd could you please suggest a refactor so that we can merge this file wtih install_rocm_magma.sh ?

Reply:
I attempted to sync-up the two versions in https://github.com/pytorch/builder/pull/1919. It tries to address the differences in the CI vs manywheel/libtorch use-case highlighted in https://github.com/pytorch/pytorch/pull/129022/files#r1671246878.